src/core/ui/keyboardNavigation/SkipLink.tsx

```diff
@@ -0,0 +1,43 @@
+import { PropsWithChildren } from 'react';
+import { Button } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import visibleOnFocus from './visibleOnFocus';
+import { Theme } from '@mui/material/styles';
+import { SystemStyleObject } from '@mui/system/styleFunctionSx/styleFunctionSx';
+
+const SKIP_LINK_Z_INDEX = 1;
+
+interface SkipLinkProps {
+  sx?: SystemStyleObject<Theme>;
+  anchor?: Element | null | (() => Element | null);
+}
+
+const SkipLink = ({ anchor, sx, children }: PropsWithChildren<SkipLinkProps>) => {
+  const { t } = useTranslation();
+
+  const handleClick = () => {
+    const anchorElement = typeof anchor === 'function' ? anchor() : anchor;
+    if (!anchorElement) {
+      return;
+    }
+    if (anchorElement.tagName === 'BUTTON' || anchorElement.tagName === 'A') {
+      (anchor as HTMLButtonElement | HTMLAnchorElement).focus();
+    }
+    // @ts-ignore
+    anchorElement.querySelector('button, [href], input, [tabindex="0"]')?.focus();
```

Comment:
```suggestion
    (anchorElement.querySelector('button, [href], input, [tabindex="0"]') as HTMLElement)?.focus?.();
```
instead of that ts-ignore?